packages/aragon-wrapper/src/cache/index.test.js

```diff
@@ -95,6 +95,10 @@ test('should observe the key\'s value for changes in the correct order', async (
   )
   // act
   const observable = instance.observe('counter', 1)
+
+  // make sure getItem is finished before emitting
+  await new Promise(resolve => setTimeout(resolve, 700))
```

Comment:
This shouldn't be needed because `getItems` finishes in ~300ms and `this.changes` starts emitting in ~500ms